persistence/src/jvmMain/kotlin/com/wire/kalium/persistence/db/UserDatabase.kt

```diff
@@ -67,9 +69,10 @@ actual fun userDatabaseDriverByPath(
     path: String,
     passphrase: UserDBSecret?,
     enableWAL: Boolean
-): SqlDriver = DriverBuilder()
-    .withWALEnabled(enableWAL)
-    .build(path)
+): SqlDriver = databaseDriver(JdbcSqliteDriver.IN_MEMORY) {
+    isWALEnabled = enableWAL
+    areForeignKeyConstraintsEnforced = true
+}
```

Comment:
Does it make sense changing the name of the method? as its state we are getting the DB driver by path and this is not true anymore? 🤔

Reply:
good catch, actually we should use it :O